### DISCUSSION TITLE
How to add columns with headers to ItemsRepeater?

### DISCUSSION THREAD

**Discussion by IAbstract** [AvaloniaUI/Avalonia#6517] [Q&A]
The documentation on `ItemsRepeater` is fairly vague. We are pointed to some Microsoft Documentation on the UWP control of the same name. So when I try to add columns with headers in a fashion that works with the UWP control, the Avalonia control doesn't like it.

I'd like to see more documentation on the control. It appears to have greater divergence from the UWP control than expected.

**Comment by maxkatz6** [comment]
Please share UWP sample that you tried to implement and how did it in Avalonia. We can't answer anything without seeing a problem.

**Reply by IAbstract** [reply]
I've partially figured out a solution. Regardless of what I was trying to implement from the UWP control, it doesn't change the fact that there is literally no documentation on the `ItemsRepeater` for Avalonia. I have to sort through quite a bit of information in the UWP docs from the 2 links you [provide on this page](https://docs.avaloniaui.net/docs/controls/itemsrepeater). When I have a full solution and time to do so, I'll post it here.
